src/StoreBase.ts

```diff
@@ -172,9 +172,10 @@ export abstract class StoreBase {
             throttledUntil: number | undefined, bypassBlock: boolean): void {
         const existingMeta = StoreBase._pendingCallbacks.get(callback);
         StoreBase._updateExistingMeta(existingMeta, throttledUntil, bypassBlock);
+
         if (existingMeta === undefined) {
             // We need to clone keys in order to prevent accidental by-ref mutations
-            StoreBase._pendingCallbacks.set(callback, { keys: _.clone(keys), throttledUntil, bypassBlock });
+            StoreBase._pendingCallbacks.set(callback, { keys: keys.slice(), throttledUntil, bypassBlock });
```

Comment:
Slice and clone have different semantics - slice makes a new array, clone copies the objects.
Does this new behaviour avoid the problem described above?

Reply:
[clone](https://github.com/lodash/lodash/blob/npm/_baseClone.js#L108) also creates a new array, just in a different way. [clone](https://github.com/lodash/lodash/blob/npm/_baseClone.js) contains many things which are useless in this case.